### PR TITLE
SNTH-23

### PR DESCRIPTION
**Pull request by GrizzlyJay**

### CODE REVIEW

File: app/src/main/java/com/dillxn/tactilesynth/MainActivity.java [hunk 1]
```diff
@@ -99,6 +151,125 @@ public void onPrepared(MediaPlayer mediaPlayer) {
 
     }
 
+    // JOSH - POPULATES DEBUG TEXTVIEWS FOR EFFECTS ON APP STARTUP
```
Comment (Dillxn) [inline app/src/main/java/com/dillxn/tactilesynth/MainActivity.java]: I do wish we had a more minimal MainActivity.java, with this and other stuff moved to other appropriate classes.

File: app/src/main/java/com/dillxn/tactilesynth/MainActivity.java [hunk 2]
```diff
@@ -99,6 +151,125 @@ public void onPrepared(MediaPlayer mediaPlayer) {
...
+        AlertDialog.Builder builder = new AlertDialog.Builder(MainActivity.this);
+
+        builder.setTitle("Select Effect");
+
+        builder.setCancelable(false);
+
+        builder.setMultiChoiceItems(effectArray, selectedEffects, new DialogInterface.OnMultiChoiceClickListener() {
+            @Override
+            public void onClick(DialogInterface dialogInterface, int i, boolean b) {
+                if (b) {
```
Comment (Dillxn) [inline app/src/main/java/com/dillxn/tactilesynth/MainActivity.java]: What does `b` here represent?

**Review by MattRas10 — APPROVED** [review]

**Review by Dillxn — CHANGES_REQUESTED** [review]
Please see comments

File: app/src/main/java/com/dillxn/tactilesynth/MainActivity.java (hunk 2 above)
Comment (GrizzlyJay) [reply app/src/main/java/com/dillxn/tactilesynth/MainActivity.java]: links the check boxes to the list of selected effects if I recall correctly.

File: app/src/main/java/com/dillxn/tactilesynth/MainActivity.java (hunk 1 above)
Comment (GrizzlyJay) [reply app/src/main/java/com/dillxn/tactilesynth/MainActivity.java]: currently working on this.

File: app/src/main/java/com/dillxn/tactilesynth/MainActivity.java (hunk 2 above)
Comment (Dillxn) [reply app/src/main/java/com/dillxn/tactilesynth/MainActivity.java]: Can we name this something clearer?

**Comment by GrizzlyJay** [comment]
Pull request outdated by progress

**Pull request closed without merging**